tests/serverpod_test_server/test_integration/diagnostics/diagnostic_events_test.dart

```diff
@@ -2,12 +2,12 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:http/http.dart' as http;
+import 'package:serverpod_test_server/src/futureCalls/test_exception_call.dart';
 import 'package:test/test.dart';
 
 import 'package:serverpod/serverpod.dart';
 
 import 'package:serverpod_test_client/serverpod_test_client.dart';
```

Comment:
Suggestion: Revert this import change, as it is now out of order.

```suggestion
import 'package:test/test.dart';

import 'package:serverpod/serverpod.dart';

import 'package:serverpod_test_client/serverpod_test_client.dart';
import 'package:serverpod_test_server/src/futureCalls/test_exception_call.dart';
```